src/layouts/MetricsLayout.tsx

```diff
@@ -37,7 +40,7 @@ export const MetricsLayout = ({
     navLinks.push({
       // Adding conditional metrics as well because we don't have yet anything to show in the metric (description, calculation, etc)
       href: `/${domain}/metrics`,
-      children: `Metrics`,
+      children: `Voting`,
```

Comment:
I am not sure of this one... because in there you are already in the Voting area and voting is split between Metrics and Ballot

Reply:
keeping this unchanges until team decides which path to go